.github/workflows/update_references.yaml

```
@@ -0,0 +1,56 @@
+name: Update references
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
+  update_references:
+
+    name: PR comment
```

```suggestion
    name: Update reference screenshots on ${{ matrix.os }}
```

Makes a little more sense here, no?